### PR TITLE
Add maisi mask aug example

### PR DESCRIPTION
**Pull request by guopengf**
Fixes # .

### Description
Add notebook example for tumor mask augmentation.

### Checks

- [ ] Avoid including large-size files in the PR.
- [ ] Clean up long text outputs from code cells in the notebook.
- [ ] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [ ] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

**Comment by guopengf** [comment]
Hi @KumoLiu, I plan to add an example mask as `tutorials/generative/maisi/data/example_mask.nii.gz`. Where should I host this file? Shall I put it into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1?

**Comment by KumoLiu** [comment]
> Hi @KumoLiu, I plan to add an example mask as `tutorials/generative/maisi/data/example_mask.nii.gz`. Where should I host this file? Shall I put it into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1?

Yes, you can host it there.

**Comment by guopengf** [comment]
I think we have this error (https://github.com/Project-MONAI/tutorials/actions/runs/9764583292/job/26953264670?pr=1748) because some utility functions have not been merged.

**Comment by guopengf** [comment]
This PR is ready to review once https://github.com/Project-MONAI/tutorials/pull/1749 is merged.

File: generative/maisi/data/mask_augmentation_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,378 @@
+{
```
Comment (KumoLiu) [inline generative/maisi/data/mask_augmentation_example.ipynb:1]: <p>Please add few sentences here to demonstrate the purpose of this notebook, thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (KumoLiu) [inline generative/maisi/data/mask_augmentation_example.ipynb:1]: <p>You can remove the output here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (guopengf) [reply generative/maisi/data/mask_augmentation_example.ipynb:1]: sure, I removed the cell output.

Comment (guopengf) [reply generative/maisi/data/mask_augmentation_example.ipynb:1]: More descriptions are added.

**Comment by KumoLiu** [comment]
@guopengf seems premerge can not pass, did you run the notebook locally?

**Comment by guopengf** [comment]
> @guopengf seems premerge can not pass, did you run the notebook locally?

Yeah, I can successfully run it locally, as you can see those cell outputs. However, the augmentation functions rely on latest monai core dev branch. I don’t if this causes problems.

**Comment by KumoLiu** [comment]
Hi @guopengf,
Could you please add the file here https://github.com/Project-MONAI/tutorials/blob/main/runner.sh#L79 to exclude max_epochs check? Thanks.
https://github.com/Project-MONAI/tutorials/actions/runs/9788615557/job/27026970836?pr=1748#step:5:610

**Comment by guopengf** [comment]
> Hi @guopengf, Could you please add the file here https://github.com/Project-MONAI/tutorials/blob/main/runner.sh#L79 to exclude max_epochs check? Thanks. https://github.com/Project-MONAI/tutorials/actions/runs/9788615557/job/27026970836?pr=1748#step:5:610

Hi @KumoLiu, thanks for the instruction! All tests are passed.

**Review by KumoLiu — APPROVED** [review]
LGTM.
If you don't have other comments, I will merge this one. cc @Nic-Ma @mingxin-zheng

**Pull request merged**